python/tk_framework_desktopserver/certificates.py

```diff
@@ -125,7 +160,7 @@ def _check_call(self, ctx, cmd):
 
         # Do not use popen.check_call because it won't redirect stderr to stdout properly
         # and it can't close stdin which causes issues in certain configurations on Windows.
-        logger.debug("%s: %s" % (ctx.capitalize(), cmd))
+        logger.info("%s: %s" % (ctx.capitalize(), cmd))
```

Comment:
can this be abit more informative? e.g. "Created certificate %s...." or something like that?